frontend/packages/kubevirt-plugin/console-extensions.json

```diff
@@ -30,5 +30,29 @@
     "flags": {
       "required": ["KUBEVIRT"]
     }
+  },
+  {
+    "type": "console.yaml-template",
+    "properties": {
+      "name": "default",
+      "model": {
+        "group": "kubevirt.io",
+        "kind": "VirtualMachine",
+        "version": "v1alpha3"
+      },
+      "template": "apiVersion: kubevirt.io/v1alpha3 kind: VirtualMachine metadata: name: vm-example labels: app: vm-example os.template.kubevirt.io/fedora31: 'true' workload.template.kubevirt.io/server: 'true' annotations: name.os.template.kubevirt.io/fedora31: Fedora 31 description: VM example spec: running: false template: metadata: labels: kubevirt.io/domain: vm-example vm.kubevirt.io/name: vm-example os.template.kubevirt.io/fedora31: 'true' workload.template.kubevirt.io/server: 'true' spec: domain: cpu: cores: 1 sockets: 1 threads: 1 devices: disks: - bootOrder: 1 disk: bus: virtio name: containerdisk - disk: bus: virtio name: cloudinitdisk interfaces: - masquerade: {} name: default networkInterfaceMultiqueue: true rng: {} resources: requests: memory: 1G hostname: vm-example networks: - name: default pod: {} terminationGracePeriodSeconds: 0 volumes: - containerDisk: image: 'kubevirt/fedora-cloud-container-disk-demo:latest' name: containerdisk - cloudInitNoCloud: userData: |- #cloud-config password: fedora chpasswd: { expire: False } name: cloudinitdisk"
```

Comment:
Ah! maybe we should use `console-extensions.yaml` instead of json lol (....hmmm....+1)

You could have used: https://onlineyamltools.com/convert-yaml-to-json

Reply:
this tool converts yaml to json, not a multiline string to a single line string with spaces/line breaks

Reply:
whoops. you're right.
So convert all new lines to `\n` and all `\` to `\\`. Then decode when you read.